### PR TITLE
Feature/e UI 8112

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8112

### Change description ###
Query list display ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-list/query-list.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="queryListData">
+  <table class="govuk-table query-list__table">
+    <caption class="govuk-table__cell query-list__caption query-list__row">
+      <div>{{ queryListData.partyName }}</div>
+    </caption>
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row query-list__row">
+      <th scope="col" class="govuk-table__header">Queries</th>
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-list/query-list.component.html:8]: @MateiHMCTS - Lines 8 to 12 need to use the Welsh translation pipe, so it should look something like this: `{{'Queries' | rpxTranslate}}`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - small change needed to incorporate Welsh translation pipe

**Review by DavidJayakumar — DISMISSED** [review]
LGTM

**Review by Daniel-Lam — DISMISSED** [review]

**Review by ronaldmansveld — CHANGES_REQUESTED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - re-approved

**Review by ronaldmansveld — APPROVED** [review]

**Pull request merged**